infra/README.md

```diff
@@ -1,13 +1,15 @@
 # Deploy
 
 To deploy you will have to have the [AWS CLI installed](http://docs.aws.amazon.com/cli/latest/userguide/installing.html#install-with-pip)
-and your [AWS credentials setup](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html). 
+and your [AWS credentials setup](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html).
+
+You will also need to have [terraform](https://www.terraform.io/downloads.html) installed.
```

Comment:
I preferred the update [here](https://github.com/wellcometrust/wellcomecollection.org/pull/40/files) as bullet points are easier to follow. What do you think?